### PR TITLE
added checkin route

### PR DESCRIPTION
**Pull request by abibou1**
john1@gmail.com and password="123456" will give you an access token.
checkin

### CODE REVIEW

File: server/src/api/v1/routes/visitors.js [hunk 1]
```diff
@@ -12,4 +14,57 @@ router.get('/admin/users', authenticatedAdmin, (req, res) => {
...
+    if (!host) {
+      res.status(404).json({ error: 'Invalid host email' });
+    } else {
+      hostID = host._id;
+    }
+  } catch (e) {
+    res.status(500).json({ error: e.message });
+  }
+  // check if visitor data is not stored, store it if it's a new visitor
+  await Visitor.findOne({ email: vEmail })
```
Comment (mazma1) [inline server/src/api/v1/routes/visitors.js:44]: You don't need the `await` keyword if you are using the `.then`, `.catch` syntax. Here's a [doc](https://davidwalsh.name/async-await) you can look at to see what the syntax for both look like

**Review by mazma1 — COMMENTED** [review]
There is an error with the review app so I couldn't test. See if you can figure out what is going on

**Comment by mazma1** [comment]
> [john1@gmail.com](mailto:john1@gmail.com) and password="123456" will give you an access token.
> checkin

Can the `visitorID` and `hostID` values in the response be plain strings instead of arrays? They are always going to have one value respectively so there is no point having them in an array.

**Comment by abibou1** [comment]
This is what the checkin looks like:

checkin1

File: server/src/models/Visit.js [hunk 2]
```diff
@@ -11,15 +11,26 @@ const VisitSchema = new Schema({
...
     type: Date,
+    default: null,
   },
 });
 
+VisitSchema.methods.toJSON = function () {
+  const obj = this.toObject();
+  delete obj.__v;
+  return obj;
+};
```
Comment (mazma1) [inline server/src/models/Visit.js:32]: Not that this is wrong, thought I shared a cleaner way of removing properties from a DB response?

File: server/src/api/v1/routes/visitors.js [hunk 3]
```diff
@@ -65,12 +67,21 @@ router.post('/admin/visitors/checkin', authenticatedAdmin, async (req, res) => {
...
     visitorID,
     hostID,
+    tag,
   });
   newVisit.save()
-    .then(() => res.status(201).json(newVisit))
+    .then(() => res.status(201).json({
+      hostID: newVisit.hostID,
+      tag: newVisit.tag,
+      timeIn: newVisit.timeIn,
```
Comment (mazma1) [inline server/src/api/v1/routes/visitors.js:82]: You need to format the value of the time so its something like `"DD-MM-YYYY, 02:30pm"` or something more readable

**Review by mazma1 — COMMENTED** [review]
Well done @abibou1, just something small left and we'll be good here

**Pull request merged**